.github/workflows/ci_packages.yml

```diff
@@ -60,4 +63,6 @@ jobs:
 
       - name: 'koch, Run CI'
         shell: bash
-        run: . ci/funs.sh && nimInternalBuildKochAndRunCI
+        run: |
+          export DYLD_LIBRARY_PATH="/usr/local/Cellar/openssl@1.1/$(ls /usr/local/Cellar/openssl@1.1 | tail -n 1)/lib:$DYLD_LIBRARY_PATH"
```

Comment:
Do will still need this? Since u linked the OpenSSL to standard `/usr/local/lib`, I guess we can remove this.

```suggestion
```